### PR TITLE
Add cluster profile parameter in install-config.yaml

### PR DESCRIPTION
**Pull request by guillaumerose**
Implementation of openshift/enhancements#200
Installer counterpart of https://github.com/openshift/cluster-version-operator/pull/404/

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
1. I do not think the profile should be a free-flowing string rather it should be a specific enum

2. I do not want to add this to the install config for all our users and only available using the create manifests and editing the clusterversion API

**Comment by wking** [comment]
> ...as we get more data around users and usecases we can think about bumping up into the API...

Where does this data come from before it goes in the ClusterVersion API?  Do you recommend adding it as a label to an existing Telemetry metric or...?

**Comment by wking** [comment]
> I do not want to add this to the install config for all our users and only available using the create manifests...

Which manifest would this belong to?  A new manifest/asset just to hold the new value?  Alternatively, could we configure this via an embarrassing, unadvertised environment variable like `_ALPHA_EXPERIMENTAL_CLUSTER_PROFILE` so CRC folks can kick the tires without having a day-0 CVO that tries to install the `default` profile manifests they'd like to exclude?

**Comment by abhinavdahiya** [comment]
Closing this as it has stalled for very long. Please feel free to re-open in case you intent to keep working on it, or consider tracking the work in an issue if this requires agreement for implementation.

/close

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -64,7 +64,8 @@ then
 		"${RELEASE_IMAGE_DIGEST}" \
 		render \
 			--output-dir=/assets/cvo-bootstrap \
-			--release-image="${RELEASE_IMAGE_DIGEST}"
+			--release-image="${RELEASE_IMAGE_DIGEST}" \
+			--cluster-profile={{.ClusterProfile}}
```
Comment (eranco74) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:68]: The CVO render gets cluster-profile parameter?
I thought it get it as env - https://github.com/openshift/cluster-version-operator/pull/483#issue-528097102

**Pull request closed without merging**